Clock/ViewTimerSet.qml

```diff
@@ -45,6 +45,13 @@ Item {
     SetButton {
         buttonTxt: "SET TIMER"
 
-        onClick: timerSet.clickTimer()
+        onClick: {
+            timerValues.isThereTimer=true;
```

Comment:
`isThereTimer` lo deve modificare `timerValues` al suo interno quando lanci `start()`. In questo modo sposti la logica in C++ invece che averla nella UI.